### DISCUSSION TITLE
using the Annotation filter

### DISCUSSION THREAD

**Discussion by Twlicker** [broadinstitute/seqr#2818] [General]
Please can someone explain the behaviour for Seqr with respect to the Annotation filter as it seems counter intuitive.

If I leave the boxes unchecked (which I would assume to mean include every and all variants) then I get a handful of variants returned.

If I check all the possible boxes under the Annotation section (at least 30 boxes, e.g. stop gains, splice variants, synonymous, intergenic etc...) then I get 1000s of variants returned.

So the question is, why is it when all the Annotation boxes are left unchecked (i.e. empty) I get so few variants, but when all 30+ Annotation boxes are checked I get 1000s of variants?

What is the default meaning for leaving all boxes unchecked?

Please can you point me to a document describing how the search features work.

Thank you

**Comment by hanars** [comment]
The annotation filter works as an "OR" filter with the pathogenicity filter. If you do not apply any pathogenicity or annotation filters, you will get variants with any annotations. If you check none of the annotations but some of the pathogenicity options, it will only return variants with the specified pathogenicity, which is what you are seeing as the handful of variants. if you check some of the pathogenicity and all of the annotations options, it will behave the same as applying no options, and return variants with any annotation again

**Comment by Twlicker** [comment]
Thanks Hanar for the quick and clear response.  That all makes sense now.

Are all the search sections working as "OR" in this fashion, or is it only the pathogenicity and annotations filters?

Is there documentation for this?

Thank you

**Comment by hanars** [comment]
This is the only such set of filters. it is documented in-app, if you read the details in the pathogenicity section
